diagnostics/tropical_pacific_sea_level/settings.jsonc

```diff
@@ -0,0 +1,55 @@
+{
+  "settings" : {
+    "driver" : "tropical_pacific_sea_level.py",
+    "long_name" : "tropical pacific sea level",
+    "realm" : "ocns",
+    "description" : "Tropical pacific sea level diagnostic",
+    "runtime_requirements": {
+        "python3" : ["matplotlib", "xarray", "cartopy","cftime","numpy"]
+    },
+    "pod_env_vars" : {
+      "Model_name" : "CESM2_CORE",
+      "Model_legend_name" : "CESM2-CORE",
+      "Model_path" : "CESM2_omip1_r1i1p1f1_gn/mon/",
+      "Model_dim0" : "time",
+      "Model_dim1" : "nlat",
+      "Model_dim2" : "nlon",
+      "Model_coord0" : "lat",
+      "Model_coord1" : "lon",
+      "zos_file" : "CESM2_omip1_r1i1p1f1_gn.zos.mon.nc",
+      "tauuo_file" : "CESM2_omip1_r1i1p1f1_gn.tauuo.mon.nc",
+      "tauvo_file" : "CESM2_omip1_r1i1p1f1_gn.tauvo.mon.nc",
+      "areacello_file" : "CESM2_omip1_r1i1p1f1_gn.areacello.mon.nc",
```

Comment:
Model data filenames should not be hard-coded; instead absolute paths to all requested variables are set via the `path_variable` setting in the new settings file format, which defaults to `<variable name>_FILE`; see [documentation](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html#varlist-entry-properties).